python/lsst/ap/association/diaForcedSource.py

```diff
@@ -121,7 +128,12 @@ def run(self,
             difference and direct images at DiaObject locations.
         """
 
-        afw_dia_objects = self._convert_from_pandas(dia_objects)
+        nDiaSources = dia_objects.loc[:, "nDiaSources"]
+        objectTable = dia_objects[nDiaSources > self.config.historyThreshold]
```

Comment:
You could do these two lines more compactly as 

`objectTable = dia_objects.query(f'nDiaSources > {self.config.historyThreshold}')`